service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -111,4 +124,14 @@ private void removeTemporaryFields(CaseDetails caseDetails, String... fields) {
             caseDetails.getData().remove(field);
         }
     }
+
+    private void publishPlacementApplicationUploadEvent(CallbackRequest callbackRequest, RequestData requestData) {
```

Comment:
do we need to pass  `RequestData requestData` here? As its already injected above and this is a method within this class